license/src/com/untangle/app/license/LicenseManagerImpl.java

```diff
@@ -497,14 +497,11 @@ public void requestTrialLicense( String appName ) throws Exception
                 if ( response != null ) { response.close(); response = null; }
             }
         } catch ( java.net.UnknownHostException e ) {
-            logger.warn("Exception requesting trial license:" + e.toString());
-            throw ( new Exception( "Unable to fetch trial license: DNS lookup failed.", e ) );
+            logger.warn("DNS failure requesting trial license:" + e.toString());
```

Comment:
i see what you're doing here, but would we want to install apps that will be unlicensed and invalid as soon as they are installed?